lib/peep.rb

```diff
@@ -0,0 +1,6 @@
+require 'active_record'
+
+class Peep < ActiveRecord::Base
+  belongs_to :user
```

Comment:
This is wild. It never occurred to me that you don't need any of the validates if you don't want them. Love it